cargo-near/src/abi/generation.rs

```diff
@@ -17,6 +17,15 @@ pub(crate) fn generate_toml(manifest_path: &CargoManifestPath) -> anyhow::Result
         .ok_or_else(|| anyhow::anyhow!("near-sdk dependency should be a table"))?
         .clone();
 
+    if !near_sdk
+        .get("features")
+        .and_then(|features| features.as_array())
+        .map(|features| features.contains(&value::Value::String("abi".to_string())))
+        .unwrap_or(false)
```

Comment:
will this fail if we ever make abi a default feat of the SDK? Assuming this is just checking the toml syntactically?

Reply:
Yes, this is just a syntactic check. I tried to find a cargo command that would tell me if a feature on one of the dependencies is enabled (even implicitly e.g. by `default`), but couldn't find one.